docs/examples/mcp-reverse-proxy/docker-compose.sse.yaml

```
@@ -0,0 +1,254 @@
+volumes:
+  postgres_data:
+  grafana_data:
+  prometheus_data:
+
+# Example environment file (.env)
+# Copy to .env and customize:
+#
+# # Database settings (TRUF.NETWORK uses kwildb/postgres image with trust auth)
+# # The kwildb/postgres image automatically creates kwild user and database
+# # Note: No password needed due to POSTGRES_HOST_AUTH_METHOD=trust
+# 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)